nano/core_test/active_transactions.cpp

```diff
@@ -662,7 +662,7 @@ TEST (active_transactions, restart_dropped)
 	}
 	// Verify the block was updated in the ledger
 	{
-		auto block (node.store.block_get (node.store.tx_begin_read (), send1->hash ()));
+		auto block (node.store.block_get (node.store.tx_begin_write (), send1->hash ()));
```

Comment:
Was changing this to a write transaction required?

Reply:
I experienced a race condition some times where the transaction for the block write had not completed before this code was reached.  This was attempting to block until the write had completed.

Reply:
ok, I wonder if it should have a comment then to prevent someone unknowingly changing it in the future, or perhaps call `node.store.block_exists` in a loop with a suitable timeout

Reply:
Good point, that's a better idiom.